### PR TITLE
Fixed failing tests on master

### PR DESCRIPTION
**Pull request by raddessi**
Since they just released 1.0 I noticed I didn't pin the repo version properly, sorry about that. This in it's current state should fix the existing tests. If I have time before you get to this I'll try to actually update the version to 1.0 as well.

### CODE REVIEW

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -133,6 +133,12 @@ def netbox_docker_repo_dirpaths(pytestconfig, git_toplevel):
             subp.check_call(
                 ["git", "fetch"], cwd=repo_fpath, stdout=subp.PIPE, stderr=subp.PIPE
             )
+            subp.check_call(
+                ["git", "reset", "--hard"],
+                cwd=repo_fpath,
+                stdout=subp.PIPE,
+                stderr=subp.PIPE,
+            )
```
Comment (raddessi) [inline tests/integration/conftest.py:141]: This was needed to fix some changes that were made when I ran black over everything.. it's probably just a good idea in general though.

**Comment by raddessi** [comment]
Hmm.. maybe this is due to memory limits. I can't get this to fail locally any more. Let me bump down the versions we test concurrently..

**Comment by raddessi** [comment]
No change there.. This is really odd. Tests continue to run fine locally.

**Comment by raddessi** [comment]
And on https://github.com/digitalocean/pynetbox/pull/327/commits/ce0c31d6ecfec24318f6fffaf0c05b5f6b78ad05 the tests only failed on 3.8.. this seems real flaky whatever it is.

**Comment by raddessi** [comment]
Woot! I have _no_ clue exactly what broke since it was building fine off these same tags before.. but updating netbox 2.10 to the netbox-docker tag 1.0.1 seems to have fixed that now. Sweeeeeeeet.

**Comment by raddessi** [comment]
Ok.. i think i know why this failed. Docker looks for the right image from a tag at time of pull but then caches that version (right? I think?) so my local version was something like 2.10.1 and passing tests but github was building against 2.10.4 and failing. Maybe there is a way to force it to check for updates.. maybe i should make ot.more clear what patch version is being tested also.

**Comment by zachmoody** [comment]
:+1: Thanks!

**Pull request merged**